packages/ui-components/src/Input/CommonInput.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import classNames from "classnames/bind";
+import "./input.module.scss";
+
+export interface CommonInputProps {
+  id?: string;
+  // this should be the div containing the input
+  fieldInput?: JSX.Element;
+  classes?: string;
+  className?: string;
```

Comment:
Why do we have both `classes` and `className` here? I feel like `classes` is an unconventional choice. Is this a hold over from the CC console component?

Reply:
className can be removed, it's not used